samples/modules/lvgl/accelerometer_chart/src/main.c

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2023 Benjamin Cabé <benjamin@zephyrproject.org>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/device.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/display.h>
+#include <zephyr/drivers/sensor.h>
+
+#include <lvgl.h>
+#include <stdio.h>
+#include <string.h>
+#include <zephyr/kernel.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(app, CONFIG_LOG_DEFAULT_LEVEL);
+
+static lv_obj_t *chart1;
+static lv_chart_series_t *ser_x;
+static lv_chart_series_t *ser_y;
+static lv_chart_series_t *ser_z;
+static lv_timer_t *sensor_timer;
+
+#define ACCEL0_NODE DT_ALIAS(accel0)
+const struct device *sensor;
+
+/* Timer handler: fetches sensor data and appends it to the chart */
+static void sensor_timer_cb(lv_timer_t *timer)
+{
+	struct sensor_value accel[3];
+	int rc = sensor_sample_fetch(sensor);
+
+	if (rc == 0) {
+		rc = sensor_channel_get(sensor, SENSOR_CHAN_ACCEL_XYZ, accel);
+	}
+	if (rc < 0) {
+		LOG_ERR("ERROR: Update failed: %d\n", rc);
```

Comment:
use `return;` here and keep the normal execution path with no indentation, makes it easier to follow the indended code path.

Reply:
+1